components/gc-subway/section-rel-info-en.html

```diff
@@ -0,0 +1,47 @@
+	"layout": "without-h1",
+	"dateModified": "2023-01-04",
+	"share": "true"
+}
+---
+
+{%- include gc-subway/docs-questions.html -%}
+
+<nav class="gc-subway provisional">
+	<h1>[Service name]</h1>
+	<ul>
+		<li><a href="#" class="active">[Page 1]</a></li>
+		<li><a href="#">[Page 2]</a></li>
+		<li><a href="#">[Page 3]</a></li>
+		<li><a href="#">[Page 4]</a></li>
+		<li><a href="#">[Page 5]</a></li>
+		<li><a href="#">[Page 6]</a></li>
+	</ul>
+</nav>
+
+<h1 property="name" id="wb-cont" class="gc-thickline">[Page 1]</h1>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam commodo elementum est, ac ultrices urna convallis vitae. Nulla nec convallis felis. Ut pretium nisl nisi. Nam gravida gravida aliquet. Morbi tincidunt lorem in purus imperdiet, id rutrum mauris sodales. Vivamus nec mattis tellus. Nunc turpis dolor, malesuada non magna nec, scelerisque tristique velit.</p>
+<p>Sed consectetur eu ligula a molestie. Vivamus convallis libero malesuada pharetra suscipit. In a pulvinar mi, quis aliquet mauris. Duis convallis nunc nunc, in euismod nisi volutpat sit amet. Integer convallis lacus non orci imperdiet, ac convallis massa mollis. Aliquam erat volutpat. Ut maximus euismod auctor. Ut ac gravida nunc. Nam non efficitur neque. Pellentesque tincidunt, libero luctus condimentum laoreet, turpis magna maximus nibh, at cursus lectus tellus in augue. Aenean scelerisque eros dui, at tincidunt eros tristique nec.</p>
+<p>Duis vel lacus elit. Mauris rhoncus urna at enim fermentum, eget fermentum est ornare. Pellentesque porttitor bibendum venenatis. Nam tempor vel felis a fringilla. Integer dui nisl, sagittis nec risus a, fermentum euismod purus. Vivamus dictum, elit quis varius eleifend, purus leo scelerisque metus, id fermentum nulla neque eget elit. Etiam vel vestibulum justo, a scelerisque eros.</p>
+<p>Nam elementum sit amet sem id finibus. Vivamus et sagittis dolor. Curabitur semper libero ac maximus viverra. Sed sed mollis nulla. Quisque ornare sapien dictum elit fermentum, ut efficitur sem placerat. Praesent nulla enim, venenatis at finibus quis, semper eu nisl. Nam finibus ante vel lacus accumsan, vitae tincidunt massa porta. Donec ullamcorper pharetra ex et dapibus. Pellentesque et viverra nibh. Pellentesque non ante sodales, interdum sapien vitae, gravida risus.</p>
+<p>Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Nunc aliquam felis massa, ut congue enim faucibus in. Vivamus rhoncus elit in pharetra bibendum. In fringilla justo non consectetur feugiat. Ut bibendum nisi lacus, eu dapibus massa semper sit amet. Fusce condimentum pretium libero eget fermentum. Duis feugiat velit at commodo accumsan. Nunc in felis leo.</p>
+
+<nav class="gc-subway-pagination">
+	<h2 class="wb-inv">Document navigation</h2>
+	<ul class="pager">
+		<li class="next"><a href="#" rel="next"><span class="wb-inv">Next: </span>[Page 2]</a></li>
+	</ul>
+</nav>
+
+<aside class="gc-subway-support">
```

Comment:
Food for thought: Do we need a special CSS class or can we leverage "pull-right" + the grid.

Reply:
This might create challenges about meaningful sequence SC